BitwardenShared/UI/Vault/Vault/VaultList/VaultListView.swift

```diff
@@ -67,6 +70,36 @@ private struct SearchableVaultListView: View {
 
     // MARK: Private Properties
 
+    /// A floating action button positioned at the bottom-right corner of the screen.
+    @ViewBuilder private var newLoginFloatingActionButton: some View {
+        VStack {
+            Spacer()
+            HStack {
+                Spacer()
```

Comment:
🎨 Instead of multiple stacks with spacers, I wonder if you could add it as an overlay to the vault's loading view? Then you wouldn't have to worry about hiding it during search and you could get the bottom trailing behavior without stacks and spacers.

```
    /// A view that displays either the my vault or empty vault interface.
    @ViewBuilder private var vault: some View {
        LoadingView(state: store.state.loadingState) { sections in
            if sections.isEmpty {
                emptyVault
            } else {
                vaultContents(with: sections)
            }
        }
        .overlay(alignment: .bottomTrailing) {
            newLoginFloatingActionButton
        }
    }
```

Also, this button can be used to create more things than logins so maybe `newItemFloatingActionButton` is better.

Reply:
yeah great call, thanks!  ill update.